experimental/FTheoryTools/src/HypersurfaceModels/constructors.jl

```diff
@@ -66,8 +70,8 @@ function hypersurface_model(base::NormalToricVariety, fiber_ambient_space::Norma
   end
   
   # Compute an ambient space
-  ambient_space = _ambient_space(base, fiber_ambient_space, D1, D2)
-  
+  ambient_space = _ambient_space(base, fiber_ambient_space, fiber_twist_divisor_classes)
+
   # Construct the model
   hypersurface_equation = generic_section(anticanonical_bundle(ambient_space))
```

Comment:
This generates a hypersurface equation with random coefficients in front of every monomial, right? Why do we want this? It doesn't seem like this would ever be used, unless I'm misunderstanding something